app/src/main/res/values/strings.xml

```diff
@@ -557,9 +557,9 @@
   <string name="resume_lesson_button">Continue</string>
   <string name="start_over_lesson_button">Start Over</string>
   <!-- Home screen greeting fragments. -->
-  <string name="home_screen_good_morning_greeting_fragment">Good morning,</string>
-  <string name="home_screen_good_afternoon_greeting_fragment">Good afternoon,</string>
-  <string name="home_screen_good_evening_greeting_fragment">Good evening,</string>
+  <string name="home_screen_good_morning_greeting_fragment">Good morning, \u0020</string>
+  <string name="home_screen_good_afternoon_greeting_fragment">Good afternoon, \u0020</string>
+  <string name="home_screen_good_evening_greeting_fragment">Good evening, \u0020</string>
```

Comment:
Could you please explain why you have added a unicode character here?

Reply:
For blank space, this was my bad approach. Now I did the same as before.